python/tvm/relax/frontend/nn/op.py

```diff
@@ -676,12 +676,31 @@ def permute_dims(x: Tensor, axes: Optional[List[int]] = None, name: str = None)
     result : Tensor
         The transposed result.
     """
+
+    # TODO(Lunderberg): This is a more extensive auto-naming than
+    # intended here.  Is this still worth it?
```

Comment:
Long-term, I want to move this automatic naming from the `nn.Module` side to the Relax side, since it could then be performed after removal of trivial bindings.  I don't expect these chains to be deep, as it only tracks trivial bindings.  The trivial binding from the Relax function parameter to the parameter's `param._expr` field should be the only one that would be tracked.